lib/cloudinary/utils.rb

```diff
@@ -383,6 +383,21 @@ def self.process_layer(layer)
   end
   private_class_method :process_layer
 
+  # Parse radius options
+  # @return [string] radius transformation string
+  # @private
+  def self.process_radius(radius)
+    if radius.is_a?(Array)
+        unless radius.length.between?(1, 4)
+          raise(CloudinaryException, "Invalid radius parameter")
+        end
+        return radius.map { |r| normalize_expression(r) }.join(":")
+    else
+      return normalize_expression(radius)
+    end
+  end
```

Comment:
This might be a bit more semantic Ruby:

```ruby
  def self.process_radius(radius)
    if radius.is_a?(Array) && !radius.length.between?(1, 4)
      raise(CloudinaryException, "Invalid radius parameter")
    end

    Array(radius).map { |r| normalize_expression(r) }.join(":")
  end
```